src/sonic-eventd/tests/eventd_ut.cpp

```diff
@@ -625,7 +632,7 @@ TEST(eventd, service)
         run_pub(mock_pub, wr_source, wr_evts);
 
         /* Published events must have been captured. Give a pause, to ensure sent. */
-        this_thread::sleep_for(chrono::milliseconds(200));
+        this_thread::sleep_for(chrono::milliseconds(1000));
```

Comment:
>1000 [](http://example.com/codeflow?start=52&length=4)

Is this a critical fix? How do you pick the magic number?

Reply:
ZMQ is async. No signal indication. It silently drops when not good.